src/less/core/description-list.less

```diff
@@ -20,6 +20,8 @@
 @description-list-line-border:                  #ddd;
 @description-list-line-color:                   #999;
 
+@description-list-term-weight:                  @base-body-font-weight;
```

Comment:
I think this should be changed to `normal`, as the value was This could break builds when you build without the base component.